src/Uno.UWPSyncGenerator/Program.cs

```diff
@@ -27,8 +27,11 @@ static void Main(string[] args)
 
 			if (mode == SyncMode || mode == AllMode)
 			{
+#if !HAS_UNO_WINUI
+				new SyncGenerator().Build(@"..\..\..\Uno.UI.Composition", "Uno.UI.Composition", "Windows.Foundation.UniversalApiContract");
+#endif
```

Comment:
This call will make bad changes to generated files, but those bad changes will be overwritten by next calls. Also the order of these statement is important (unfortunately) by the approach I took.